dotnet-roslyn/config/Audacia.CodeAnalysis/Audacia.CodeAnalysis.csproj

```diff
@@ -19,7 +19,7 @@
     <PackageReference Include="IDisposableAnalyzers" Version="4.0.2" />
     <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="6.0.0" />
     <PackageReference Include="Roslynator.Analyzers" Version="4.0.2" />
-    <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.435" />
+    <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.556" />
```

Comment:
Is there a reason we are using a beta version? 

The latest stable version is `1.1.118`, could we use this?

Reply:
`1.1.118` was released on 29/04/2019 - all the updates since then have been beta versions, not sure why

Reply:
Do we need the updates?

Reply:
I've specifically updated now because some of the rules in the older version don't know about newer C# features so give incorrect warnings - see this issue I raised: https://github.com/audaciaconsulting/Audacia.CodeAnalysis/issues/19

The very fist version of this Audacia.CodeAnalysis used StyleCop `1.2.0-beta.354` so I don't think being on beta version is of concern, especially as they are not seeming to release non-beta versions.

Reply:
Fair play will resolve.

Reply:
Just want to add onto this why they are all beta versions - will try to document this elsewhere at some point.

Beta because they have not audited all the new C# features in 8, 9, 10 and 11 - https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3500

Beta is not an issue because it is a Dev-Only dependency - https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3420#issuecomment-1058215862